### PR TITLE
Notes for the exclude predicted

### PR DESCRIPTION
**Pull request by olaaustine**
[ENSVAR-6372](https://www.ebi.ac.uk/panda/jira/browse/ENSVAR-6372)

### CODE REVIEW

File: docs/htdocs/info/docs/tools/vep/script/vep_other.html [hunk 1]
```diff
@@ -437,7 +437,7 @@ <h4>Input</h4>
   <h2 id="refseq">RefSeq transcripts</h2>
   
 
-  <p>If you prefer to exclude predicted RefSeq transcripts (those with identifiers beginning with "XM_" or "XR_") use <a href="vep_options.html#opt_exclude_predicted">--exclude_predicted</a>.</p>
+  <p>If you prefer to exclude predicted RefSeq transcripts (those with identifiers beginning with "XM_" or "XR_") use <a href="vep_options.html#opt_exclude_predicted">--exclude_predicted</a>.<br>Only available for human on the GRCh38 assembly</p>
```
Comment (nuno-agostinho) [inline docs/htdocs/info/docs/tools/vep/script/vep_other.html:440]:
```suggestion
  <p>If you prefer to exclude predicted RefSeq transcripts (those with identifiers beginning with "XM_" or "XR_") use <a href="vep_options.html#opt_exclude_predicted">--exclude_predicted</a>.<br>Only available for human on the GRCh38 assembly.</p>
```

**Review by nuno-agostinho — CHANGES_REQUESTED** [review]

**Review by nuno-agostinho — APPROVED** [review]
Looks good to me, thanks @olaaustine!

**Comment by sarahhunt** [comment]
@olaaustine - this isn't quite right. The option works to remove predicted RefSeq transcipts for all species (dog, mouse, etc) but we don't support predicted RefSeq transcipts for GRCh37.

We could document that here
http://www.ensembl.org/info/docs/tools/vep/script/vep_other.html#refseq

**Comment by sarahhunt** [comment]
Looks good - thanks Ola!

**Pull request merged**